### DISCUSSION TITLE
taro3 加nutui 很多组件异常

### DISCUSSION THREAD

**Discussion by yishen-yishen** [NervJS/taro#9833] [Q & A] [answered]
"@tarojs/taro": "3.3.0",
"@nutui/nutui-taro": "^3.1.1",
两个应该都是最新版本了。
Picker、DatePicker、Swiper等都无法编译。
小程序开发工具中提示
`[Vue warn]: Failed to resolve component: picker `

~~~~~~~~~~~1
使用过nutui，仓库中的来编译。结果同样是这些组件异常。但是前段时间编程成功过一次，猜测是环境问题或者是版本问题。
有没有遇到过的，给个思路啊？

**Comment by honlyHuang** [comment, marked as answer]
你 `taro init` 的时候直接选 taro nutui 的模板咯

**Reply by yishen-yishen** [reply]
没有用，也试过。
猜测可能是自己的环境问题。或者版本问题。换其他的了。

**Comment by holyrong** [comment]
我的NutUI 3.1.14也有这个问题，请问怎么回事？
@Chen-jj

**Reply by zxqzy** [reply]
这个问题现在解决了吗

**Comment by holyrong** [comment]
![image](https://user-images.githubusercontent.com/8318475/149120273-76435273-8a96-489c-a965-43694ee30eae.png)
![image](https://user-images.githubusercontent.com/8318475/149120336-540994fb-69e6-4f41-945d-adceb27364c0.png)

**Comment by Chen-jj** [comment]
vue 的提示很清晰了，配置一下 vueLoader 即可.https://docs.taro.zone/docs/vue3#vueloaderoption
